v1/summary.yaml

```diff
@@ -27,7 +27,7 @@ paths:
         sentences, as well as information about a thumbnail that represents
         the page.
 
-        Stability: [experimental](https://www.mediawiki.org/wiki/API_versioning#Experimental)
+        Stability: [stable](https://www.mediawiki.org/wiki/API_versioning#Stable)
```

Comment:
Hm, with all of the tickets filed by the reading team, I think this deserves to be `unstable` ?